apps/music/js/db.js

```diff
@@ -108,6 +108,12 @@ function initDB() {
       PlayerView.stop();
     }
 
+    // Dismiss the keyboard if it's active, by blurring all the input elements.
+    var inputs = document.getElementsByTagName('input');
+    for (var i = 0; i < inputs.length; i++) {
+      inputs[i].blur();
+    }
```

Comment:
@jimporter Make sense and thanks Jim! I will update the patch as you said and set review to you.